crates/block-producer/src/test_utils/mod.rs

```diff
@@ -45,7 +43,7 @@ impl Random {
         AccountIdBuilder::new().build_with_rng(&mut self.0)
     }
 
-    pub fn draw_digest(&mut self) -> Digest {
-        self.0.draw_word().into()
+    pub fn draw_word(&mut self) -> Word {
```

Comment:
Just worth checking whether we want these (and similar) fns to remain using the digest word. Unsure myself for this one.